astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -866,22 +866,22 @@ def test_rdb(read_rdb):
     assert table["B"].dtype.kind in ("S", "U")
     assert_equal(table["C"].dtype.kind, "f")
 
+    text = "A\tB\tC\nN\tS\tN\n4\tb\ta"  # C column contains non-numeric data
     with pytest.raises(ValueError) as e:
```

Comment:
```suggestion
    # C column contains non-numeric data
    with pytest.raises(ValueError) as e:
        read_rdb("A\tB\tC\nN\tS\tN\n4\tb\ta")
```
Agreed; temporary variables are unnecessary here, and so is forcing the line split with trailing comma – the comment could better be placed on a separate line (be it inside or outside the context, dunno how picky PT012 is there).